xml/System.Buffers.Text/Utf8Parser.xml

```diff
@@ -51,13 +51,24 @@
         <Parameter Name="standardFormat" Type="System.Char" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="bytesConsumed">To be added.</param>
-        <param name="standardFormat">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">The Utf8 string to parse.</param>
+        <param name="value">When the method returns, contains the value parsed from <paramref name="source" />, if the parsing operation succeeded. </param>
+        <param name="bytesConsumed">If the parsing operation was successful, contains the length in bytes of the parsed substring when the method returns. If the method fails, <paramref name="bytesConsumed" /> is set to 0.</param>
```

Comment:
I get that the source xml comments had this issue, but "substring" seems odd to me when we are not talking about strings. Maybe use "slice" since we are talking about spans.

Also, maybe "in bytes" should be in parenthesis or comma separated. Adding "when the method returns" also seems redundant to me, making the description quite verbose and hard to read.

`If the parsing operation was successful, contains the length, in bytes, of the parsed slice.`

Reply:
Maybe we can mirror the wording from the TryFormat (bytesWritten):
```C#
<param name="bytesWritten">When the method returns, contains the length of the formatted text in bytes.</param>
```
```text
If the parsing operation was successful, contains the length of the parsed text in bytes. If the method fails, <paramref name="bytesConsumed" /> is set to 0.
```